doc/sphinx/methods.rst

```diff
@@ -126,7 +126,7 @@ The Mulliken class also can take a progress class as an argument so that the pro
     m.calculate()
 
 .. index::
-    single: methods; Löwdin Population Analysis
+    single: methods; L��wdin Population Analysis
```

Comment:
Something went wrong with the character here

Reply:
This is weird -- I've corrected this. Thanks!